log/log.go

```diff
@@ -1,72 +1,105 @@
 package log
 
 import (
-	"fmt"
+	"os"
```

Comment:
Add "strings" to get the `ToLower` function
```suggestion
	"os"
	"strings"
```